### ISSUE TITLE
Bower packages breaks the rightside moment-viewer

### ISSUE THREAD

**Issue by haviduck** [IBM/watson-multimedia-analyzer#39]
Doing a straight bower install, the bower packages will break the moment viewer. Launch an instance, start the gui and click "Show moment" will throw this error: 

`angular.js:15018 TypeError: Failed to set the 'currentTime' property on 'HTMLMediaElement': The provided double value is non-finite.
    at Object.seekTime (videogular.js:405)
    at VideoController.js:66
    at Scope.$broadcast (angular.js:19165)
    at MomentController.vm.gotoMoment (watson.moment.directive.js:68)`

This is confirmed on win 10 and ubuntu 18.04 + 16.04.

**Comment by haviduck** [comment]
Turns out its a well known issue after moving over to AMI api keys. visualrecognition.js v3 doesn't look like it copes with it.

**Comment by rhagarty** [comment]
@haviduck - this code pattern has known issues when using new instances of Visual Recognition that require IAM api keys. There are also some API changes in v3 that break the way we process images.

Hopefully, someone will be able to address these issues shortly.

**Comment by haviduck** [comment]
@rhagarty - yeah, took me some deep diving to figure it all out. i'm very close to a fix - i just gotta find whatever is STILL triggering the old faces model.  its being logged and im finding it hard to figure out how to update the Faces GA. when done ill commit and u can check it out

**Comment by haviduck** [comment]
nah. its not all about the faces.. that is sorted - but they removed the endtime's on speechtotext. if you use speakers that'll give the endtime per scene, so working on that.
